cpp/tests/unary/unary_ops_test.cu

```diff
@@ -0,0 +1,147 @@
+/*
+ * Copyright (c) 2019, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <cudf/cudf.h>
+#include <tests/utilities/base_fixture.hpp>
+#include <cudf/unary.hpp>
+#include <cudf/column/column_factories.hpp>
+#include <tests/utilities/column_utilities.hpp>
+#include <cudf/utilities/type_dispatcher.hpp>
+#include <tests/utilities/type_lists.hpp>
+#include <tests/utilities/column_wrapper.hpp>
+#include <cudf/legacy/interop.hpp>
+#include <tests/utilities/legacy/cudf_test_utils.cuh>
+#include <vector>
+
+template <typename T>
+cudf::test::fixed_width_column_wrapper<T> create_fixed_columns(cudf::size_type start, cudf::size_type size, cudf::bool8 nullable) {
```

Comment:
You can/should still use `bool` for function arguments like this. You just need to use `bool8` when interacting with columns of type `BOOL8`.
```suggestion
cudf::test::fixed_width_column_wrapper<T> create_fixed_columns(cudf::size_type start, cudf::size_type size, bool nullable) {
```

Reply:
ahh, I did replace all, my bad.